tests/VObject/Component/VCardTest.php

```diff
@@ -92,6 +92,14 @@ public function validateData()
             ],
             "BEGIN:VCARD\r\nVERSION:4.0\r\nUID:foo\r\nORG:Acme Co.\r\nFN:Acme Co.\r\nEND:VCARD\r\n",
         ];
+        // No FN, NICKNAME fallback
+        $tests[] = [
+            "BEGIN:VCARD\r\nVERSION:4.0\r\nUID:foo\r\NICKNAME:JohnDoe\r\nEND:VCARD\r\n",
```

Comment:
Note: CI did not run in this PR. This line was missing an "n"
`UID:foo\r\NICKNAME:JohnDoe` needs to be `UID:foo\r\nNICKNAME:JohnDoe`

Fixed in #576